pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
```

Comment:
@FRidh: my 'problem' was that the package is both: a Python dkim library by origin, but a set of very useful tools with man pages and all as well. I could move this to development/python-modules, so other Python applications can easily use it?